### PR TITLE
Sponsorship policy

### PR DESCRIPTION
**Pull request by Folyd**
I have a brief discussion with @ZhangHanDong about our sponsorship policy, including OpenCollective tiers, author remuneration, and editor salary. All of these policies are drafts, we need you reviewing before we reach an agreement. Any comments are welcome.

### CODE REVIEW

File: pages/sponsor.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+Each author must submit an expense request on the OpenCollective platform. After the admin approves the request, the Fiscal Host will make a payout.
+
+## How Rust Magazine pay the editors?
+
+```callout, theme:red
+This is a **draft** and is subject to change in the future.
+```
+
+Rust Magazine has several editors who take turns being responsible for each Issue. The editor on duty for each Issue will receive a payment of **$50**. If an editor is not on duty for a particular Issue, Rust Magazine will not pay them for that Issue.
```
Comment (Xuanwo) [inline pages/sponsor.md:35]: Do we need to make it clear that every issue will have only one editor?

**Review by Xuanwo — COMMENTED** [review]
Mostly LGTM. I hope RustMagazine can become more sustainable.

File: pages/sponsor.md (hunk 1 above)
Comment (Folyd) [reply pages/sponsor.md]: Fixed.

**Review by Xuanwo — APPROVED** [review]

**Review by LCrossman — COMMENTED** [review]
The idea behind this is great and it sounds really exciting!, I like the idea that everyone can be paid for their work.  I have a couple of questions, what happens if some authors need to be paid but insufficient sponsors have come forwards - Who is responsible? 
 Also, maybe covering the costs should be the first thing - is the list authors, editors, costs in a particular order?  Last thing, what if the editor on duty is not the best editor for a particular article?  An article might be much better suited to a specific editor.
Minor point - the subheadings could be slightly better worded.

**Comment by Folyd** [comment]
Good questions, thanks @LCrossman.

- > what happens if some authors need to be paid but insufficient sponsors have come forwards - Who is responsible?

We can't pay the author if our balance is insufficient, however, the author can still submit an expense request on OpenCollective, after we have enough sponsors, we'll approve it and make a payout. So no one else needs additional responsibility for this.

- > Also, maybe covering the costs should be the first thing - is the list authors, editors, costs in a particular order?

There is no particular order, I agree to put covering the costs first though.

- > what if the editor on duty is not the best editor for a particular article? 

This isn't a problem. Editor on duty isn't mean that the editor is responsible all thing, it means he/she is in charge of this issue. When he/she meets something he/she is not good at it, just seek help from the editor team.

**Pull request merged**